docs/en/installing.rst

```diff
@@ -122,31 +133,19 @@ Create a new project and select one of the available boards. You can change afte
    board = esp32-c3-devkitm-1
    framework = arduino
 
-How to update to the latest code
-********************************
+How to update to the stable code based on 3.0.x 
+-----------------------------------------------
 
-To test the latest Arduino ESP32, you need to change your project *platformio.ini* accordingly.
-The following configuration uses the upstream version of the Espressif development platform and the latest Arduino core directly from the Espressif GitHub repository:
+To use the actual Arduino ESP32, you need to change your project *platformio.ini* accordingly.
+The following configuration uses the community version pioarduino and the latest Arduino core from the pioarduino GitHub repository:
```

Comment:
```suggestion
To use the latest supported Arduino ESP32, you need to change your project *platformio.ini* accordingly.
The following configuration uses the community version ``pioarduino`` and the latest Arduino core from the ``pioarduino`` GitHub repository:
```

Reply:
Changed, and corrected the github source for Arduino 3.0.x release -> fetched from the release section of this repo